configs/speedSections.yml

```diff
@@ -0,0 +1,6 @@
+doof:

```

Comment:
Should the speedSections be subsection in the bot's config itself? ie the doof speedSections can be in doof.yml and minibot.yml etc? This means less files and more complete config for a specific bot.